app/static/src/app/components/options/TagInput.vue

```diff
@@ -20,7 +20,12 @@ export default defineComponent({
         VueTagsInput
     },
     props: {
-        tags: Array as PropType<Tag[] | null>
+        tags: Array as PropType<Tag[] | null>,
+        numericOnly: {
+            type: Boolean,
```

Comment:
is there a difference between Boolean and bool? Or am I getting confused with different languages?

Reply:
It's how Vue defines the types of its props, not quite the same as the Typescript types